proto/generated-go/v1/actuator_service.pb.gw.go

```diff
@@ -155,6 +155,7 @@ func local_request_ActuatorService_GetResourcePackage_0(ctx context.Context, mar
 // UnaryRPC     :call ActuatorServiceServer directly.
 // StreamingRPC :currently unsupported pending https://github.com/grpc/grpc-go/issues/906.
 // Note that using this registration option will cause many gRPC library features to stop working. Consider using RegisterActuatorServiceHandlerFromEndpoint instead.
+// GRPC interceptors will not work for this type of registration. To use interceptors, you must use the "runtime.WithMiddlewares" option in the "runtime.NewServeMux" call.
```

Comment:
Looks like you need to upgrade your local buf?

Reply:
Maybe buf upgraded their protoc-gen-go-grpc.